www_src/lib/api.js

```diff
@@ -86,6 +86,39 @@ api.AUTHENTICATE_URI = api.BASE_LOGIN_URI + '/login/oauth/access_token';
 api.SIGN_UP_URI = api.BASE_LOGIN_URI + '/create-user';
 api.USER_URI = api.BASE_LOGIN_URI + '/user';
 
+api.validateOptions = function (name, requiredFields, options) {
+
+  requiredFields = requiredFields || [];
+
+  if (typeof name !== 'string' || requiredFields instanceof Array === false) {
```

Comment:
why `instanceof` instead of testing for what it needs to support, like foreach or slice?